### ISSUE TITLE
Static files( frontend ) should be served by an NGINX server

### ISSUE THREAD

**Issue by agentmilindu** [scorelab/OpenDF#52]
Serving static files ( HTMLs, CSS, JS and image file of the front-end ) with NGINX server is good IMO. Once the front-end is updated, the files could be merged, minified, versioned and put in a 'dist' directory. We then can served these files though NGINX.

**Comment by saisankargochhayat** [comment]
I want to work on this. Can i be assigned to this task. Thank you.

**Comment by agentmilindu** [comment]
Hi @saisankargochhayat, I forgot to mention that this task is for GSoCers ( a small part of a large GSoC task this is actually. http://www.scorelab.org/gsoc-2017/#opendfkubernetize ). If you like, you can work on this whole GSoC task! :D ( of course, only if you are interested in GSoC )

**Comment by saisankargochhayat** [comment]
Yes , i am interested in GSoC , can you direct me to a small task to test the waters . I really like the idea outlines this (Kubernetize OpenDF) and would like to work on this or "Image annotation and describe module". Thank you for leading the way :)
